code/modules/reagents/Chemistry-Goon-420BlazeIt.dm

```diff
@@ -502,8 +502,9 @@ datum/reagent/hotline/addiction_act_stage4(var/mob/living/M as mob)
 	M.adjustBrainLoss(rand(1,30))
 	M.hallucination += 30
 	M.druggy = max(M.druggy, 30)
-	if(prob(1))
-		M.visible_message("<span class = 'userdanger'>[M] clutches at their chest! It looks like they're having a heart attack!</span>")
-		M.adjustBruteLoss(80) // don't do drugs kids
+	var/mob/living/carbon/human/H = M
+	if(prob(1) && !H.heart_attack)
+		H.visible_message("<span class = 'userdanger'>[H] clutches at their chest! It looks like they're having a heart attack!</span>")
```

Comment:
userdanger span is for danger to the USER. We use it for notifying people that they've been attacked, etc. It shouldn't be used in a public message.